{
  "type": "class",
  "name": "DefaultUncaughtExceptionHandlerWithName",
  "qualified_name": "net.minecraft.DefaultUncaughtExceptionHandlerWithName",
  "package": "net.minecraft",
  "modifiers": [
    "public",
    "class"
  ],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [],
    "inheritance_tree": [
      "java.lang.Object"
    ]
  },
  "nested_classes": [],
  "fields": [
    {
      "name": "logger",
      "modifiers": [
        "private",
        "final"
      ],
      "type": "org.apache.logging.log4j.Logger",
      "description": "",
      "deprecated": false
    }
  ],
  "constructors": [
    {
      "name": "DefaultUncaughtExceptionHandlerWithName",
      "modifiers": [
        "public"
      ],
      "parameters": [
        {
          "name": "p_131801_",
          "type": "org.apache.logging.log4j.Logger",
          "annotations": []
        }
      ],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [
    {
      "name": "uncaughtException",
      "modifiers": [
        "public"
      ],
      "return_type": "void",
      "parameters": [
        {
          "name": "p_131803_",
          "type": "Thread",
          "annotations": []
        },
        {
          "name": "p_131804_",
          "type": "Throwable",
          "annotations": []
        }
      ],
      "description": "",
      "deprecated": false
    }
  ],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "Frameworks/1.17.1/net/minecraft/DefaultUncaughtExceptionHandlerWithName.html"
}